src/listener/can_open_listener.py

```diff
@@ -86,6 +82,15 @@ def listen_to_network(self):
                     logging.error(f'The requested sdo ({hex(sdo_index)})'
                                   ' is not received!')
 
+    async def async_network_loop(self):
+        ''' Function to loop the through all nodes asynchronously.
+            Loops forever.
```

Comment:
willen we dit? Niet ergens een exit condition?

Reply:
Je wilt toch dat het altijd naar het netwerk luistert?